### PR TITLE
Add initial README.md

### PR DESCRIPTION
**Pull request by vaporos**
Adds basic project information into README.md.

Signed-off-by: Shawn T. Amundson <amundson@bitwise.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## How to Participate
+
+We welcome contributors, both organizations and individuals, to help shape
+project direction, contribute ideas, provide use cases, and work on specific
+tools and examples.
+
+## More Information
+
+- [Rust Crate and Documentation](https://crates.io/crates/transact)
+- [Hyperledger Transact mailing list](https://lists.hyperledger.org/t/transact)
```
Comment (agunde406) [inline README.md]: This link should be https://lists.hyperledger.org/g/transact

Comment (vaporos) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## How to Participate
+
+We welcome contributors, both organizations and individuals, to help shape
+project direction, contribute ideas, provide use cases, and work on specific
+tools and examples.
+
+## More Information
+
+- [Rust Crate and Documentation](https://crates.io/crates/transact)
```
[the remark this answers is not in the document]
Comment (vaporos) [reply README.md]: Fixed

**Comment by vaporos** [comment]
Added readme entry to Cargo.toml.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- The [transact-rfcs](https://github.com/bitwiseio/transact-rfcs) repository
+  contains RFCs (requests for comments) for proposed and approved changes to
+  Hyperledger Transact.
+
+## How to Participate
+
+We welcome contributors, both organizations and individuals, to help shape
+project direction, contribute ideas, provide use cases, and work on specific
+tools and examples.
```
[the remark this answers is not in the document]
Comment (vaporos) [reply README.md]: Fixed.

**Pull request merged**